### DISCUSSION TITLE
Thrifting hotspots in the area

### DISCUSSION THREAD

**Discussion by spiritedgoldfish** [katskillskat/katskills-org#7] [General]
As material items grow in expense, thrifting is a solid way to cut costs as well as decrease the amount of waste we put into landfills. While shopping isn't something I generally enjoy, I tend to find the task less daunting when it comes to thrifting. Commercial stores tend to show what clothing, furniture, and knick-knacks you _should_ be buying whereas, in places like this, it feels like less of the dealer's choice and more of a catch-all. The number of antique stores in the region is a driving force for tourism, but tourism prices can come with it. Thrift and consignment shops take items of all calibers and ages and put them back into the community, whereas previously, the items would have been a part of someone's dump run. There are several shops in the area that I'm aware of and visit on occasion as well as the driving force for this post, which I only saw for the first time, recently. 
While in the market for a couch, I was told that the Oneonta Reuse Center was a great place to find any kind of used furniture you could want, at fairly reasonable prices. What I found was one and a half warehouses stacked and packed to the brim with anything you could think of. Vinyl records, tapes, phone cases, fine china, hutches, couches, cameras, books, art, pots, and pans are all well stocked and well priced. Whether it's the most tattered copy of Moby Dick you have ever seen or a pristine collectors edition of Lord of the Rings on VHS, these items have or at least have had value to someone and had plenty of value leftover in most cases. I was absolutely delighted walking through mazes and personally disappointed that I hadn't found this place sooner. 
A personal favorite store of mine would be Shakedown Street, also located in Oneonta. This consignment shop holds a massive amount of clothing that is both gender and size-inclusive. They also sell for local brands, businesses, and artists in the area. Shakedown Street added a cafe in the back of the store that I, as a fairly picky eater, have never been dissatisfied with. While you would pay more for a t-shirt there than you may at Salvation Army, the quality is incomparable as the store only takes items they feel capable of reselling. I consider it a go-to store for clothing staples at a great price. 
Lastly, in a town teeming with antique stores, there is Alice's Attic in Walton. Another thrift/consignment shop that has great finds and good pricing. A little digging for something you'd genuinely enjoy feels much more reasonable compared to shopping from commercial racks that all mimic the others. Both this store and the Oneonta Reuse center, hire employee's from the ARC. While I'm aware there are positive kickbacks for the companies that do this, it is an extensive and tedious process to uphold and the fact that they're doing it is deeply important to our community members with disabilities. 
There are shops that I'm aware of such as Silk and Treasures in (surprise, surprise) Oneonta and The Green Giraffe in Unadilla that I have yet to visit personally. Still, these stores and ones similar, serve a great purpose of access on many levels and deserved a little shout-out, in my opinion. 
What are your favorite shops in the area? Do you have any other thrifting honey-holes that I may not have mentioned? Do you personally thrift or have an interest in it?

**Comment by kendra-lacey** [comment]
The Margaretville Auxiliary Thrift Shop has a lot of great stuff, they just aren’t open that often probably due to the fact that it is completely run by volunteers. ( I believe they are in need of them) 
It’s open from 12-3:30 Wednesday’s & Saturday’s 

Pine Hill Community Center has their own little thrift shop as well 
Open Monday 11:00am-3:00pm, Wednesday-Friday 11:0am-3:00pm, Saturday and Sunday 11:00am-4:00pm
*not %100 sure on the pill hill thrift shop hours I’m going to check in to get the correct info 

I also feel like there should be a way to exchange or give away clothes for free 
Sometimes I purge my closet and I have a bag of clothes sitting around forever
I want to give it directly to someone who needs it..not donate it to a thrift shop so they can sell it, or drop it in a bin and then they get all smelly and who really knows where those bins go. 
There’s a free trade site on Facebook but.. not everyone has Facebook & let’s “face” it Facebook is slightly corrupt and is not ideal. 

Do you think there’s an answer that doesn’t involve Facebook where we can keep it local & safe ?

**Comment by spiritedgoldfish** [comment]
Thank you for the recommendations, I will absolutely check those places out!
Direct donation is something I prefer as well, it's always nice to swap with a friend who shares your style.  When I do donate to the big orange bins, I know that whatever happens to it afterward is out of my control and that I can only donate with the intention of helping people who need it. 
Facebook was the first site in quite a while that got a lot of people onto the internet/social media. While I'm not personally a fan of Facebook either, it tends to be the most widely used platform amongst a large age range. I agree that it isn't ideal however, the accessibility is there. Even those without Facebook tend to have a relative or friend post their items for them in my experience. 
I think that a community/ town website with information on places to give and receive donations would be extremely beneficial. Also, locally organized clothing swaps similar to the Gender Affirming Clothing swap that was held in Oneonta some time ago would be huge for many communities. 
A separate site serving the purpose that the Facebook pages serve would need a lot of hype and consistent backing from the community. Great items would have to be posted often to keep people engaged. It would require a great deal of consistent community outreach to get the link and purpose out there. In short, I think the answer is out there, it would just be quite a bit of work for whoever decides to tackle the project.
